packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -113,10 +124,8 @@ describe("wrangler", () => {
           result: false,
         }
       );
-      const { warnings } = await runWrangler("init");
-      expect(warnings).toContain("wrangler.toml file already exists!");
-      const parsed = TOML.parse(await fsp.readFile("./wrangler.toml", "utf-8"));
-      expect(typeof parsed.compatibility_date).toBe("string");
```

Comment:
Do we not still want to check this?

Reply:
imo, `wrangler init` shouldn't test the _validity_ of an existing `wrangler.toml`, that's for other commands that actually read it. We can revisit this ofc.

Reply:
Oh hmm, good point. I did a test for this in the previous one, I'll do it here too.